mobile/app.json

```diff
@@ -3,7 +3,7 @@
     "name": "dsocial",
     "slug": "dsocial",
     "platforms": ["ios", "android"],
-    "version": "1.0.2",
+    "version": "1.0.4",
```

Comment:
This version number is not used by semantic release. Instead, it's used when you compile the app to release on stores. We may automatize the process to increment the version.